alt-ergo.opam.template

```diff
@@ -6,3 +6,7 @@ license: [
   "LicenseRef-OCamlpro-Non-Commercial"
   "Apache-2.0"
 ]
+
+conflicts: [
+  "result" {< "1.5"}
+]
```

Comment:
Dependencies should be specified in the `dune-project` (here with a `(conflicts)` field in the package declaration, see https://dune.readthedocs.io/en/stable/dune-files.html#stanza-package ), not in the template file.

Reply:
Thanks! I didn't find the right stanza.